backend/database/database.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+
+const host = process.env.host || 'localhost';
+
+const connection = () =>
+  mongoose.connect(`http://${host}:27017/data`, () => {
+    console.log(`database connected at ${host}`);
```

Comment:
never use console.log in production...eslint should have caught this error also

use console.warn, console.info, or console.error

Reply:
kk ill set up lint for that too